.gitignore

```diff
@@ -1,4 +1,3 @@
+__pycache__
+.mypy_cache
 .python-version
-secrets
```

Comment:
Should `secrets` remain to align with the instructions to create such a directory in the README?